docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# H2 How to Contribute:
```

Comment:
```suggestion
## How to Contribute:
```

Then number of `#` characters determines the header level. Check out [ATX Headings in the spec](https://github.github.com/gfm/#atx-headings) for examples.